backend/src/schema/resolvers/user_management.js

```diff
@@ -15,40 +15,29 @@ export default {
     },
   },
   Mutation: {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-03T18:29:45Z" title="Wednesday, July 3rd 2019, 8:29:45 pm +02:00">Jul 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -5,50 +5,39 @@ import { neo4jgraphql } from 'neo4j-graphql-js'
 
 export default {
   Query: {
-    isLoggedIn: (parent, args, { driver, user }) => {
+    isLoggedIn: (_, args, { driver, user }) => {
       return Boolean(user && user.id)
     },
     currentUser: async (object, params, ctx, resolveInfo) => {
       const { user } = ctx
       if (!user) return null
-      return neo4jgraphql(object, { id: user.id }, ctx, resolveInfo, false)
+      return neo4jgraphql(object, { id: user.id }, ctx, resolveInfo, true)
```


I think we want to change this back to false, no?